rfc_process.md

```diff
@@ -0,0 +1,85 @@
+* ACCEPTED: The abstract interface is versioned to reflect the final changes.
+
+# Proposal Components
+
+## RFC (Request for Change)
+Proposals begin as a single document, written in markdown and committed to the `/rfc` directory.
+
+An RFC consists of the following:
+
+* Proposal name, status, and author.
+* Problem statement.
+* Historical background.
+* Abstract interface.
+* Use cases.
+* Risk Assessment.
+
+An RFC has the following qualities:
+
+* The issue is clearly defined, and within the scope of the OpenTracing charter.
+* The utility to the OT community is illustrated with concrete use cases. It should be clear not only what will change, but how the new interface is expected to be used.
+* Changes to the abstract interface are clearly defined. Language-specific interfaces will be implemented in a later phase, but the abstract interface must be clear enough that the changes could be ported to many languages without much deviation.
+* All expected behavior should be documented, in sufficient detail such that language-specific tests and examples can be generated from the descriptions.
+* Risks, backwards compatibility, and potential for inconsistency are addressed.
+
+## Tracking Issue
+Often, a proposal will consist of multiple artifacts, such as GitHub issues, pull requests, release candidates, etc. In order to manage these resources, a single, long-running tracking issue is assigned to each proposal.
+
+A tracking issue consists of the following:
+
+* Proposal name, status, and author.
+* Short summary or proposal abstract.
+* A link to the RFC.
+* Lists of open issues, PRs, and upcoming deadlines.
+* Links to relevant artifacts, such as release candidates.
+* Links to meetings and other discussion channels.
+
+## Release Candidates
+Each language interface implements the abstract interface contained in the RFC.
+
+A complete release candidate consists of the following:
+* Repository branch containing the new version.
+* Packages for installing the new version.
+* Tracers which bind to the new version.
+* Major instrumentation ported to the new version.
+
+# Proposal Lifecycle
+
+## Draft
+* Pull request to add the draft RFC to `/rfc`. 
+* Tracking issue for the proposal is opened.
+* Debate continues via pull requests against the draft.
+
+A draft proposal is first submitted as a pull request against the specification repository. 
+
+Once there is consensus that the pull request meets the minimum criteria for the beginnings of a draft, and has strong potential to be accepted, the pull request is merged. A tracking issue is created at this point to manage the proposal. The draft RFC is then refined via more pull requests.
+
+Consensus is used to determine that all of the abstract proposal work has been completed, and the investigation should move to language implementations.
+
+## Test
+* RFC status changed from `Draft` to `Test`.
+* Release Candidate per major language is created.
```

Comment:
Do we have a list of "major languages" defined somewhere?

Reply:
good point, "supported language" is a better term.